orchagent/portsorch.cpp

```diff
@@ -5666,6 +5764,20 @@ void PortsOrch::updatePortOperStatus(Port &port, sai_port_oper_status_t status)
     if (port.m_type == Port::PHY)
     {
         updateDbPortOperStatus(port, status);
+        if (port.m_autoneg > 0)
+        {
+            if (status == SAI_PORT_OPER_STATUS_UP)
+            {
+                updatePortStateAutoNeg(port);
+                m_port_state_poll[port.m_alias] &= ~PORT_STATE_POLL_AN;
+            }
+            else
```

Comment:
Should we make sure that port is not admin_down before enabling it for polling?

Reply:
Yes, will do.

Reply:
Done, the poller will skip the operation if the port is admin down